lib/commands/execute.js

```diff
@@ -24,23 +24,26 @@ const NATIVE_COMMANDS_MAPPING = {
   queryAppState: 'macosQueryAppState',
 
   appleScript: 'macosExecAppleScript',
+
+  startRecordingScreen: 'startRecordingScreen',
```

Comment:
Does it make more sense to have macos prefix like `macosStartRecordingScreen` if followed other namings?

Reply:
Base Appium driver declares an endpoint for this function, so I would rather keep it as is. It would make it easier for clients to add wrappers there, because we can call this API as an extension, but also as a "standard" API